colmena/proxy.py

```diff
@@ -1,18 +1,65 @@
 """Utilities for interacting with ProxyStore"""
 import logging
 import warnings
-import proxystore as ps
 
-from typing import Any, Union, List
+import proxystore
+from proxystore.store.endpoint import EndpointStore
```

Comment:
I feel like you changed to one import per line for a reason, and figure that the reason's a good one. What's the store here?

Reply:
Nice. Sounds like a good practice.